src/main/resources/i18n.properties

```diff
@@ -37,3 +37,9 @@ gui.assembly.successful=Assembly successful.
 gui.assembly.failed.with.0.errors=Assembly failed with {0} errors:\n\n
 runtime.error.0.at.1=Runtime error: "{0}" at {1}
 internal.error.0=Internal error:{0}
+new.file=New file (Ctrl+N)
+open.file=Open file (Ctrl+O)
+save.file=Save file (Ctrl+S)
+assemble.program=Assemble program (Ctrl+B)
+step.instruction=Step instruction (F11)
+run.program=Run program (F5)
```

Comment:
The shortcuts are not portable, as there is no `ctrl` on macOS. The best thing is to remove them, as this should be managed in a proper menu that is still missing.

Also, I would group all these with a leading `gui.` to show what these properties are about.

Reply:
I didnt know about that